x-pack/filebeat/Jenkinsfile.yml

```diff
@@ -113,6 +113,18 @@ stages:
         labels:
           - "aws"
       stage: extended
+    cloudAWS:
+        cloudAWS: "mage build test"
+        withModule: true       ## run the ITs only if the changeset affects a specific module.
+        dirs:                  ## run the cloud tests for the given modules.
+          - "x-pack/filebeat/input/awss3/_meta/terraform"
+        when:                  ## Override the top-level when.
+          changeset:           ## when PR contains any of those entries in the changeset
+            - "^x-pack/filebeat/input/awss3/.*"
```

Comment:
@v1v `- x-pack/libbeat/common/aws` could impact as well, I see you added folder only related to the specific beat.

Reply:
Yes, I added a comment in the description mentioned I didn't enable the cross-beat dependencies for this stage.
It could be added anyway

Reply:
I missed it, sorry :)

Reply:
No worries, worth to mention that forgetting!! Thanks for your reviews 🙏

Reply:
I just enabled it :)